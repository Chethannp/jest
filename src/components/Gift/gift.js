import React, { Component } from 'react'

class Gift extends Component {
    render() {
        return (
            <div>
                <h2>Gift Givers</h2>
            </div>
        )
    }
}

export default Gift
