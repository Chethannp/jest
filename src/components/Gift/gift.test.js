import React from "react";
import Gift from "./gift";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

test("matches the snapshot", () => {
  const wrapper = shallow(<Gift />);
});
