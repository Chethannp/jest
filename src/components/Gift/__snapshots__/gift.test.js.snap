// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div>
  <h2>
    Gift Givers
  </h2>
</div>
`;
